src/app/components/feed/FeedTopBar.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { createFragmentContainer, graphql } from 'react-relay/compat';
+import { browserHistory } from 'react-router';
+import IconButton from '@material-ui/core/IconButton';
+import ShareOutlinedIcon from '@material-ui/icons/ShareOutlined';
+import TeamAvatar from '../team/TeamAvatar';
+import Can from '../Can';
+import styles from './FeedTopBar.module.css';
+
+const FeedTopBar = ({ team, feed }) => {
+  const hasList = Boolean(feed.saved_search);
+
+  const handleClick = () => {
+    browserHistory.push(`/${team.slug}/list/${feed.saved_search.dbid}`);
+  };
+
+  if (!feed.published) {
+    return null;
+  }
+
+  return (
+    <div className={`${styles.feedTopBar} feed-top-bar`}>
+      <TeamAvatar team={{ avatar: team.avatar, slug: team.slug }} size="24px" />
+      <div className="typography-body2">
+        {
+          hasList ?
+            <div className="feed-top-bar-list">
+              <span className={styles.feedListTitle}>{feed.saved_search.title}</span>
+              <Can permissions={feed.permissions} permission="update Feed">
+                <IconButton size="small" onClick={handleClick} className={styles.feedListIcon}><ShareOutlinedIcon style={{ fontSize: 12 }} /></IconButton>
```

Comment:
please change this to 16px, just to get the icon a bit larger

Reply:
Done here: https://github.com/meedan/check-web/commit/d45bfc0253b418cd0293b1bd1267300dde646709